web/src/main/webapp/libs/app.bundle.min.js

```diff
@@ -116732,7 +116742,7 @@
             }, "Please try again with the latest version of ", v.a.createElement("a", {
                 className: "unsupported-desktop-browser__link",
                 href: "https://www.google.com/chrome/"
-            }, "Chrome"), " ", this._showFirefox() && v.a.createElement(v.a.Fragment, null, "or ", v.a.createElement("a", {
+            }, "Chrome"), " ", this._showFirefox() && v.a.createElement(v.a.Fragment, null, "or ", v.a.createElement("a", {
```

Comment:
Unnecessary change, sorry.
I think the mistake is caused by the character encoding of the editor.